BootloaderCommonPkg/Include/Library/FirmwareUpdateLib.h

```diff
@@ -20,29 +20,38 @@ WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
 #include <IndustryStandard/Acpi30.h>
 #include <Guid/SystemResourceTable.h>
 
-#define CMOS_ADDREG     0x70
-#define CMOS_DATAREG    0x71
+#define CMOS_ADDREG             0x70
+#define CMOS_DATAREG            0x71
 
-#define MAX_UPDATE_REGIONS   4
+#define MAX_UPDATE_REGIONS      4
 
-#define FW_UPDATE_SM_INIT     0xFF
-#define FW_UPDATE_SM_PART_A   0xFE
-#define FW_UPDATE_SM_PART_B   0xFD
-#define FW_UPDATE_SM_PART_AB  0xFC
+#define FW_UPDATE_SM_INIT             0xFF
+#define FW_UPDATE_SM_CAP_PROCESSING   0x7F
+#define FW_UPDATE_SM_PART_A           0x7E
+#define FW_UPDATE_SM_PART_B           0x7D
+#define FW_UPDATE_SM_PART_AB          0x7C
 
-#define RTC_PORT_ID 0xC3
-#define PCH_LPC_REG_P2SB_UNHIDE 0xE1
-#define PCH_LPC_REG_P2SB_BAR 0x10
+#define FW_UPDATE_IMAGE_UPDATE_NONE         0xFF
+#define FW_UPDATE_IMAGE_UPDATE_PENDING      0xFE
+#define FW_UPDATE_IMAGE_UPDATE_PROCESSING   0xFC
+#define FW_UPDATE_IMAGE_UPDATE_DONE         0xF8
+
+#define FW_UPDATE_PARTITION_A   0
+#define FW_UPDATE_PARTITION_B   1
+
+#define FW_UPDATE_SIG_LENGTH    256
 
-#define FW_UPDATE_PARTITION_A 0
-#define FW_UPDATE_PARTITION_B 1
+#define RTC_PORT_ID             0xC3
+#define PCH_LPC_REG_P2SB_UNHIDE 0xE1
+#define PCH_LPC_REG_P2SB_BAR    0x10
```

Comment:
Line 44 to 46  is not related with firmware update, should be removed from this header file.

Reply:
Removed them